lib/happo/public/happo-runner.js

```diff
@@ -199,6 +199,11 @@ window.happo = {
       this.getFullRectRecursive(node.children[i], box);
     }
 
+    // Cut off things that render off the screen to the top or left, since those
+    // won't be in the screenshot file that we then crop from.
+    box.left = Math.max(box.left, 0);
+    box.top = Math.max(box.top, 0);
```

Comment:
Yeah. If this wasn't the behavior, I think a few of our current happo examples at brigade would fail. 

I'll add a comment. Probably after merging this PR to master though.